zebedee/client_test.go

```diff
@@ -4,6 +4,7 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"io/ioutil"
```

Comment:
(fyi): just note that ioutil is deprecated from go 1.17 and is moved to the io package. So an update of the go version requires updating this to io.

Reply:
Thanks. I'll replace with `os.ReadFile(…)` as per deprecation notes in https://pkg.go.dev/io/ioutil#ReadFile